### ISSUE TITLE
Computed properties are not calculated on rehydration

### ISSUE THREAD

**Issue by rmariuzzo** [ctrlplusb/easy-peasy#590]
Hey there! I'm still loving this library, thank you! 😊 

Not sure if that's the expected behavior. I noticed that when rehydrate a state computed props doesn't get recalculated.

![demo](https://user-images.githubusercontent.com/439839/97512795-cc4bb300-1960-11eb-808c-bbbadd247cf7.gif)

You can see a demo to reproduce it [here](https://codesandbox.io/s/persisted-computed-props-in-easy-peasy-mgxtj?file=/src/store.ts).
---
```
    "easy-peasy": "4.0.1",
    "react": "16.13.1",
    "react-dom": "16.13.1",
```

**Comment by mighdoll** [comment]
I see: if you add a todo or two in your demo example, and then reload the browser window, the count of todos resets to zero.
 
Perhaps try using easy-peasy 4.1.0-beta.4. That seems to fix it.

**Comment by rmariuzzo** [comment]
@mighdoll I can confirm this issue is fixed using version `4.1.0-alpha.7` or higher. Ty!

**Comment by ctrlplusb** [comment]
💜

Hopefully will have a stable release coming up.
